tests/test_maths.py

```diff
@@ -215,7 +215,9 @@ class TestCombination(unittest.TestCase):
     def test_combination(self):
         self.assertEqual(10, combination(5, 2))
         self.assertEqual(252, combination(10, 5))
-        
+    def test_combination_memo(self):
+        self.assertEqual(10272278170, combination_memo(50, 10, {}))
```

Comment:
We don't want to change the API, and passing an empty dictionary doesn't look good. We need an outside wrapper to make its API same as `combination`